pkg/networkmanager/networking.css

```
@@ -39,14 +39,14 @@
         padding: 0;
     }
 
-    .pf-c-card {
-        overflow-x: auto;
-    }
-
     #network-interface-hw, #network-interface-mac {
       font-size: var(--pf-global--FontSize--md);
       font-weight: var(--pf-global--FontWeight--normal);
     }
+
+    #network-interface-status {
```

It's a class, so shouldn't it be:

```suggestion
    .network-interface-status {
```

...?

---

indeed.